src/main/java/kirby/commands/DeadlineCommand.java

```diff
@@ -0,0 +1,52 @@
+package kirby.commands;
+
+import kirby.tasks.Deadline;
+import kirby.TaskList;
+import kirby.Ui;
+import kirby.Storage;
+import kirby.exceptions.KirbyMissingArgumentException;
+import java.io.IOException;
+
+/**
+ * DeadlineCommand class handles the command to create a Deadline task.
+ */
+public class DeadlineCommand extends Command {
+    private String inputString;
+
+    /**
+     * Constructor for the class DeadlineCommand.
+     *
+     * @param inputString arguments of a command.
+     */
+    public DeadlineCommand(String inputString) {
+        this.inputString = inputString;
+    }
+
+    /**
+     * {@inheritDoc}
+     * Creates a Deadline task if arguments are valid.
+     */
```

Comment:
I like that you provided comments for all public methods. Good news is that prof damith has recently removed the requirement of needing javadoc comments for overriding methods.
https://se-education.org/guides/conventions/java/intermediate.html
> Write descriptive header comments for all public classes/methods.
>
>You MUST write header comments for all classes, public methods. But they can be omitted for the following cases:
>i. Getters/setters
>ii. When overriding methods (provided the parent method's Javadoc applies exactly as is to the overridden method)